test/specs/string-checker.js

```diff
@@ -45,6 +45,15 @@ describe('modules/string-checker', function() {
             });
         });
 
+        it('should not check empty string (#1354)', function() {
+            var spy = sinon.spy(StringChecker.prototype, '_checkJsFile');
```

Comment:
Another not great solution is to assert that none of the rules are called. 

I'm fine with this if you make it protected (we don't have the notion of friend functions) for testing (jsdoc tag change should suffice, comment is a bonus).